src/contributor-guide/contributing.md

```diff
@@ -248,6 +248,78 @@ Admins access will be shared via comment on GitHub.
 -  Magento EngCom Team
 -  [Contributor]({{ site.baseurl }}/contributor-guide/contributors.html)
 
+### Customize deployed instances {#customize-deployed-instances}
+
+In some cases a custom environment is necessary in order to test an issue, or a pull request, to achieve such a need there are a set of commands that, combined with the [Deploy vanilla Magento instance](#vanilla-pr) or [Deploy instance based on PR changes](#deploy-pr), will allow you to customize your deployed instance environment.
+
+#### Magento Edition:
+
+By appending the text below to [Deploy vanilla Magento instance](#vanilla-pr) or [Deploy instance based on PR changes](#deploy-pr) commands you will be able to specify which edition you would like to have deployed.
+
+```text
+with edition {$edition}
+```
+
+For `edition`, the supported values are `ee` to deploy Magento Commerce edition or `b2b` to deploy the Magento Commerce with B2B modules.
+
+#### Environment:
+
+By appending the text below to [Deploy vanilla Magento instance](#vanilla-pr) or [Deploy instance based on PR changes](#deploy-pr) commands you will be able to specify different versions of the applications/services running on the instance.
+
+```text
+with env[ironment] PHP {$phpVersion}, search-engine ElasticSearch {$searchEngineVersion}, database {$dbEngine} {$dbEngineVersion}
```

Comment:
should this be:

with env PHP {$phpVersion}, ....

Reply:
In this case it can be either:

`with env` or `with environment`

Maybe we should just leave the short form of it.